website/docs/opengl/c1/2-hello-quad.md

```diff
@@ -0,0 +1,140 @@
+
+Let's draw something on-screen! In this tutorial, you'll learn:
+
+* How to initialize OpenGL.
+* How to clear the window.
+* What vertex array objects, vertex buffers, and element array buffers are, and how to use them.
+* How to create and use shaders.
+* How the OpenGL pipeline works.
+
+This tutorial will feature a lot of content and new info, so we'll take it slow and with lots of explanations.
+
+## Initializing OpenGL
+The first thing we need to do before we can issue OpenGL commands, is to initialize it. Fortunately, Silk.NET windowing does most of the hard part for us, and we only have to get the GL api.
+
+Add the following using directive to the top of your file, you'll need this to use OpenGL:
+
+```cs
+using Silk.NET.OpenGL;
+```
+
+Then, at the top of your class, add the following:
+
+```cs
+private static GL _gl;
+```
+
+Finally, in your `OnLoad` method, add the following:
+
+```cs
+_gl = _window.CreateOpenGL();
+```
+
+What are we doing here? Silk.NET requires you to keep a **reference** to the OpenGL API. If you've used or seen OpenGL in C, you'll notice that this is different to the way that it is done there. This is done so that you can more easily keep track of multiple contexts. If you don't know what that is, don't worry about it for now, we won't be using it in these tutorials.
+
+<?# Info "If you take a look at the source in the Silk.NET samples repository, you will notice that it uses `_gl = GL.GetApi(_window)`. This is another way to get the GL API, however when using Silk.NET windowing, it's recommended that you use `_window.CreateOpenGL()` instead." /?>
+
+Now, run your application again. If all is good, you should see no change. Awesome! Let's do our first steps in OpenGL: Clearing the window.
+
+## Clearing the window
+Before we start this, let's take a look at what makes up an OpenGL window. 
+
+A window consists of several **framebuffers**. A framebuffer is essentially a texture that can be drawn to by OpenGL. An OpenGL window will consist of the following framebuffers:
+
+* Color buffer
+* Depth stencil buffer
+
+On top of this, a window will often contain at least two of these buffers. This is known as **double-buffering** and is imperitive for VSync to work properly. One buffer is displayed, while another is rendered to. 
+
+This is what makes up the **swapchain**, which, if you've ever looked into Direct3D, you should be familliar with. OpenGL does not use the term, nor does it allow you to manage this yourself, however it is still in the background, and it is helpful to know this.
+
+Now that we know this, we can get on with clearing the window.
+
+Add the following using directive to the top of your file:
+
+```cs
+using System.Drawing;
+```
+
+Then, in your `OnLoad` method, underneath `GetApi()`, add the following:
+
+```cs
+_gl.ClearColor(Color.CornflowerBlue);
+```
```

Comment:
Silk.NET is a 1:1 binding of OpenGL, and as a result the names are mapped 1:1. Please consult the OpenGL documentation for any clarifications on the bindings themselves.

(...and we're certainly not going to rename anything within 2.X as that'd be massively breaking!)

Reply:
Might be worth suggesting on 3.0 in one of the meetings though.

Reply:
Can't imagine adding it into 3.0 is a good idea, as again that would go against the 1:1 bindings, and would certainly annoy plenty of users, myself included.

If some people wanted this type of functionality, perhaps a small library could be written over the GL functions to do this behaviour instead, and the developer could apply to have it as part of SilkCommunity (I'm not sure what the guidelines are there..). Just an idea, I personally won't be developing anything like that since `_gl.ClearColor = Color.CornflowerBlue;` is no real different to `_gl.ClearColor(Color.CornflowerBlue)`, so imo the change is a bit pointless.